include/identity.php

```diff
@@ -656,8 +656,10 @@ function advanced_profile(App $a) {
 			$profile['birthday'] = array( t('Birthday:'), $val);
 
 		}
-
-		if ($age = age($a->profile['dob'],$a->profile['timezone'],'')) {
+		if ($a->profile['dob']
```

Comment:
Well, it would, I guess; I chose this just because it looked like the preferred way to do it, according to everything else in this file. Should I change it?